ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -3242,7 +3257,13 @@ MmLoadSystemImage(IN PUNICODE_STRING FileName,
     /* FIXME: Call driver verifier's loader function */
 
     /* Write-protect the system image */
+#if !defined(CORE_16449_IS_FIXED) && DBG
+    DPRINT1("(CORE-16449) Calling MiWriteProtectSystemImage() for %wZ (%wZ)\n",
```

Comment:
Full name is enough, no need to print the base name. I'd rather print the image base.

Reply:
Changed.